Project1/PathMap.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+#include "ID.h"
+#include "CellPath.h"
+#include <map>
+
+
+//TODO(Torrey): rename this class
+template <typename grid, typename cell>
+class Path_Map :
+	private id_cost_map
+{
```

Comment:
`class Path_Map : private id_cost_map {`

Please, sir. Newlines don't grow on trees around here.

Reply:
The Lorax is sad

Reply:
2 lines is one less than 3, but still twice as many as 1.

Reply:
that is outdated westy